src/foam/nanos/crunch/CapabilityJunctionStatus.js

```diff
@@ -10,31 +10,31 @@ foam.ENUM({
     {
       name: 'PENDING',
       label: { en: 'pending', pt: 'pendente'},
-      documentation: `This is awaiting verification. Capability access may take up to 24hrs.`,
+      documentation: `Our team is currently reviewing. Approvals may take up to 24 hours.`,
```

Comment:
Use regular single-quotes: ' instead of back-quotes ` when not using interpolation or multiple-lines since it parses faster and has less chance for unintended consequences.

Reply:
Ah okay. changing it.